site/guide/set-up-model-workflows.qmd

```diff
@@ -0,0 +1,213 @@
+date: last-modified
+---
+
+Workflows can be configured to match your organizational needs for overseeing model development, validation, or implementation activities.
+
+Default workflows are provided for you as suggestions. You can customize the following default workflows:
+
+- **Model documentation:** When documentation is created
+- **Inventory model:** When the model is created
+
+<!-- 
+- Model validation workflow is in missing as of May 2024
+- These default workflows will eventually be replaced by a trigger you can define
+ -->
+
+ Setting up your workflow involves two major steps:
+
+ 1. [Configuring workflow steps](#configure-workflow-steps) 
+ 2. [Linking the workflow together](#link-workflow-together)
+
+
+ <!-- The following is no longer true as of May 31, 2024: -->
+
+<!-- ::: {.callout-warning title="Adjustments to workflows are not retroactive."}
+When a workflow is edited, the new flow will only apply to new models created after the adjustment. 
+::: -->
+
+## Prerequisites
+
+- To customize workflows, you must hold the `Customer Admin` or `Developer` role
+- [Resource statuses](customize-resource-statuses.qmd) must be fully configured
+
+## Configure workflow steps
+
+To add or edit workflow steps:
+
+1. [Log in to the {{< var vm_platform >}}]({{< var vm_url >}}).
+
+2. In the left sidebar, click [**{{< fa gear >}} Settings**]({{< var vm_url >}}/settings).
+
+3. Under Workplace Settings, select **{{< fa arrow-right-arrow-left >}} Workflows**.
+
+4. Under available workflows, select the process you'd like to add or edit the steps for:
+ 
+    - To add a step, drag and drop a new step onto the canvas. Hover over the step until the **{{< fa ellipsis-vertical >}}** menu appears, then click on it and select **{{< fa gear >}} Configure**.
+    - To edit a step, hover over the step until the **{{< fa ellipsis-vertical >}}** menu appears. Then click on it and select **{{< fa gear >}} Configure**.
+
+5. After you're finished with step configuration, click **Save** to apply your changes before closing the configuration panel with **{{< fa x >}}**. 
+
+    ![](configure-steps.gif){fig-alt="A gif workflow step configuration."}
+
+    After you've configured a step, you can then [link your workflow together](#link-workflow-together).
+
+::: {.callout-tip appearance="minimal"}
+Refer to **[Available step types](#available-step-types)** for information on available step types and details on how to set up conditional requirements.
+:::
+
+### Available step types
+
+#### Status Change
+
+- Transitions a status. 
+- Requires [available resource statuses](customize-resource-statuses.qmd) associated with the workflow.
+
+| Field | Description |
+|---:|---|
+| When these conditions are met (optional) | [Add conditional requirements](#add-conditional-requirements) to qualify for this step.  |
+| Set status to (required) | Select the status to transition to. |
+: Status Change step configuration {.hover tbl-colwidths="[40,60]"}
```

Comment:
```suggestion
: **Status Change** step configuration {.hover tbl-colwidths="[40,60]"}
```

GLOBAL: References to UI elements you interact with should be bolded.

Do you know what the icon library is that we're using? We might be able to add the same to Quarto.

Reply:
Also, does `{.hover tbl-colwidths="[40,60]"}` work for you? It doesn't seem to do anything when I tested locally.

Reply:
> Do you know what the icon library is that we're using? We might be able to add the same to Quarto.

I asked Jeffrey about the icons, they're not from Font Awesome (they're little images from Figma). I figured it was more trouble than it was worth to add images here.

> Also, does {.hover tbl-colwidths="[40,60]"} work for you? It doesn't seem to do anything when I tested locally.

It does, it adjusts the width effectively for me, not sure why it doesn't for you. [This is also how the official Quarto docs tell you to assign table widths.](https://quarto.org/docs/authoring/tables.html#column-widths)

Reply:
> It does, it adjusts the width effectively for me, not sure why it doesn't for you. [This is also how the official Quarto docs tell you to assign table widths.](https://quarto.org/docs/authoring/tables.html#column-widths)

I figured it out — I'd never used `.hover` before but mainly because **I try to avoid tables wherever possible**.
Maybe my comments relates to our use of tables more generally — they can be problematic for screen readers and require special attention to be accessible ([How tables can make content less accessible](https://medium.com/@lizzie.g.bruce/accessibility-and-tables-c233f6b2b823)) 

In this case, I do think tables are helpful but they should only be used to represent data or parameters. Tables should not be used to format or group other content together. And even where a table might make sense, I would use a simple definition list before using a table, as it sidesteps accessibility issues altogether.

Reply:
That makes sense. Would you like me to change these to a list, or can we keep them as tables? I think the information here makes *sense* as a table, as they are key / value pairs, but I understand the argument against them.

Reply:
I'd leave as is, but let's avoid a proliferation of tables as we expand our docs. "Use tables sparingly" would likely be a good addition to our style guide at some point.